packages/web/src/features/clubDetails/frames/ClubDetailMainFrame.tsx

```diff
@@ -47,6 +51,13 @@ const ResisterInfoWrapper = styled.div`
   align-items: center;
 `;
 
+const ResponsiveWrapper = styled(FlexWrapper)`
+  gap: 60px;
+  @media (max-width: ${({ theme }) => theme.responsive.BREAKPOINT.sm}) {
```

Comment:
```suggestion
  @media (max-width: ${({ theme }) => theme.responsive.BREAKPOINT.md}) {
```
#663 이슈에서, 721~960px 구간의 gap을 40px로 맞추는 작업 진행 중입니다. 동아리 상세 페이지의 경우 이 페이지의 반응형 구현이 진행되고 있는 이 PR에서 수정되는 게 좋을 것 같아 리뷰 남깁니다!

Reply:
어차피 나중에 머지하는 사람이 merge conflict 해결하게 될 것 같아서 진현이가 하던 둘 다 하던 상관은 없을 것 같은뎅

Reply:
요거 먼저 머지하고, 진현이가 나중에 머지하면서 확인만 해주면 될 것 같아서 굳이 뺄? 필요까진 ㅇ없을 것 같긴 합니당!
어케 생각행 @jinhyeonkwon @wjeongchoi

Reply:
저는 좋아요~~

Reply:
@jinhyeonkwon change requested를 할 경우 그 사람의 approve를 받아야만 머지 가능해서 정말 critical한 사항이 아니라면 comment로 리뷰 남겨줘! (그래서 너가 어프루브를 해야한다는 뜻)

Reply:
구럼 가은이가 어차피 해놨으니까 진현이 피알에서 이 페이지에 해당하는 변경사항 빼도 될듯!

Reply:
아 이미 해둔 걸 확인 못했네
감사합니다ㅎㅎ